recipes/blis/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+export BLIS_COMPILER="$CC"
+{{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
This won't work as `{{ PYTHON }}` is a jinja2 variable which will only be expanded in the recipe.
```suggestion
"$PYTHON" -m pip install . --no-deps -vv
```